storage/blockchain/src/config.rs

```diff
@@ -59,8 +62,10 @@ pub use cuprate_database_service::ReaderThreads;
 #[derive(Debug, Clone, PartialEq, PartialOrd)]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 pub struct ConfigBuilder {
+    network: Network,
+
     /// [`Config::db_directory`].
-    db_directory: Option<Cow<'static, Path>>,
+    db_directory: Option<PathBuf>,
```

Comment:
I also made changes to the blockchain & txpool database so we can handle switching networks more easily.

Reply:
I don't think this needs to be changed from `Cow<'static, Path>` -> `PathBuf`.

Reply:
It doesn't _need_ to but `Cow<'static, Path>` is more annoying to work with for little benefit, it's literally 0 benefit now as we need runtime paths.